ldraw_to_scad

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+""" Translate LDraw library or file to OpenSCAD library or file. """
+
+import os
+import argparse
+from ldraw_to_scad import LDrawConverter
+
+
+def translatedir(converter, src, dest, selfcontained=False):
+    """ translate a whole model directory """
+    types = ['.mpd', '.ldr', '.dat']
+    lst = {}
+    for fdir, _, files in os.walk(src, followlinks=True):
+        rel = os.path.relpath(fdir, src)
+        for file in sorted(files):
+            base, ext = os.path.splitext(file)
+            key = os.path.join(rel, base)
+            if ext in types:
+                if key in lst:
+                    print(f'Skipping {os.path.join(src, key+lst[key])}')
```

Comment:
This prints skipping, but still sets `lst[key]` - doesn't quite sound right to me.

Reply:
Note that this is the case where two files with identical file names but different extensions (like foo.mpd and foo.ldr) exist causing a collision in the target file, both resolving to foo.scad. Therefore we skip the one with lower priority (foo.ldr in the example) and store the extension of the _new_ file with higher priority. I acknowledge though that this likely deserves some extra comment in the code. ;)

Reply:
Comment added.